docs/src/integrations/exchange.md

```diff
@@ -721,6 +721,10 @@ must then be used to determine the effective balance change. These fields will
 identify the token mint and account owner (main wallet address) of the affected
 account.
 
+Note that if a receiving account is created during the transaction, it will have no
+`preTokenBalance` entry as there is no data to decode the state from.  In this
```

Comment:
```suggestion
`preTokenBalance` entry as there is no existing account state.  In this
```
Abstract away a little more of the implementation?

Reply:
whoops mergify beat me here.  Good idea though. I'll add it to a follow up